File: Entities/Models/Employee.cs
namespace Entities.Models
{
    public class Employee
    {
        public Guid Id { get; set; }
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public string? EmployeeNumber { get; set; }
        public string? Position { get; set; }
        public string? Email { get; set; }
        public List<DeviceAssignment>? DeviceAssignments { get; set; }
    }

}

File: InventryUI/src/app/_interface/employee.ts
export interface Employee {
    id: string;
    firstName: string;
    lastName: string;
    employeeNumber: string;
    position: string;
    email: string;
  }
  
  export interface EmployeeForCreationDto {
    firstName: string;
    lastName: string;
    employeeNumber: string;
    position: string;
    email: string;
  }
  